### PR TITLE
Add pages from appliance to k8s for datahub and databroker 

### PR DESCRIPTION
**Pull request by TzviLipschitz**
https://github.com/SoftwareAG/c8y-docs/pull/2070/files [into 2024 branch](https://github.com/SoftwareAG/c8y-docs/commit/21b5f205e04c86dcb5d9164d4e26f4b9b7f59dc8)

### CODE REVIEW

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Select the **Remember me** checkbox if you want the browser to remember your cre
 
 Finally, click **Login** to enter Edge. Initially, you will be taken to the [Cockpit](/cockpit/cockpit-introduction/) application, if not configured differently. For further information about the {{< product-c8y-iot >}} standard applications see [Available applications](/get-familiar-with-the-ui/available-applications/).
 
-![image alt text](/images/users-guide/cockpit/cockpit-home-screen.png)
+![Cockpit home screen](/images/users-guide/cockpit/cockpit-home-screen.png)
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:120]: @TzviLipschitz We do not use the term "screen" in UI documentation. Please refer to our style guide for the correct equivalent. Eventhough, this is an alt text, we need to be consistent.

File: content/edge-kubernetes/k8-edge-configuration.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Configuring Edge
+weight: 55
+layout: bundle
+section:
+  - edge_server
+---
+
+Learn how to configure many different aspects of {{< product-c8y-iot >}} Edge.
```
Comment (MWindel) [inline content/edge-kubernetes/k8-edge-configuration.md:9]:
```suggestion
Learn how to configure many different aspects of Edge.
```

File: content/edge/edge-installation-bundle/accessing-platform.md [hunk 3]
```diff
@@ -123,7 +123,7 @@ Select the **Remember me** checkbox if you want the browser to remember your cre
 
 Finally, click **Login** to enter the Edge appliance. Initially, you will be taken to the [Cockpit](/cockpit/cockpit-introduction/) application (if not configured differently).
 
-![image alt text](/images/users-guide/cockpit/cockpit-home-screen.png)
+![Cockpit home screen](/images/users-guide/cockpit/cockpit-home-screen.png)
```
Comment (MWindel) [inline content/edge/edge-installation-bundle/accessing-platform.md:126]: @TzviLipschitz see similar comment above.

File: content/edge/using-edge-bundle/branding.md [hunk 4]
```diff
@@ -4,27 +4,12 @@ title: Branding
 layout: redirect
 ---
 
-With the Branding feature, you can fully customize the look of your tenant to your own preferences.
+With the Branding feature, you can fully customize the look of your tenant to your own preferences. To access the branding feature:
```
Comment (MWindel) [inline content/edge/using-edge-bundle/branding.md:7]: spelling of "Branding feature" is not consistent. Please use one way of spelling.

File: content/edge-kubernetes/k8-using-edge-bundle/branding.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+weight: 55
+title: Branding
+layout: redirect
+---
+
+With the Branding feature, you can fully customize the look of your tenant to your own preferences. To access the branding feature:
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/branding.md:7]: spelling of "Branding feature" is not consistent. Please use one way of spelling.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md [hunk 6]
```diff
@@ -87,6 +87,6 @@ Events:
 …………
   Normal   Installing        2m     cumulocityiotedge  finished installing thin-edge
 …………
-  Normal   Ready             1m     cumulocityiotedge  Cumulocity IoT Edge installation is complete, and it's now running version 1017.0.0-4638
+  Normal   Ready             1m     cumulocityiotedge  Cumulocity IoT Edge installation is complete, and it's now running version {{< c8y-edge-current-version >}}.0.0-XXXX
```
Comment (MWindel) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md:90]: add shortcode

File: content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md (hunk 1 above)
Comment (TzviLipschitz) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/accessing-edge.md:120]: @MWindel This change was originally made by Diana, I was just copying it across. Will replace with page as needed.

File: content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md (hunk 6 above)
Comment (TzviLipschitz) [reply content/edge-kubernetes/installing-edge-on-k8-bundle/install-edge.md:90]: @MWindel No shortcode needed. This is code and will always say Cumulocity IoT Edge

**Review by MWindel — APPROVED** [review]

**Review by bbyreddy — APPROVED** [review]

**Pull request merged**